akka-http-tests/src/test/scala/akka/http/scaladsl/unmarshalling/MultipartUnmarshallersSpec.scala

```diff
@@ -275,6 +275,17 @@ class MultipartUnmarshallersSpec extends FreeSpec with Matchers with BeforeAndAf
                        |--XYZABC--""".stripMarginWithNewline("\r\n")))).to[Multipart.FormData] should haveParts(
           Multipart.FormData.BodyPart.Strict("email", HttpEntity(`application/octet-stream`, ByteString("test@there.com"))))
       }
+      "with one element and name value in quotes" in {
+        Unmarshal(HttpEntity(
+          `multipart/form-data` withBoundary "XYZABC" withCharset `UTF-8`,
```

Comment:
I saw your comment after fixing. I made it little bit different but should be ok :)